### PR TITLE
Rocko Update

### PR DESCRIPTION
**Pull request by telphan**
Required by #51

### CODE REVIEW

**Comment by agherzan** [comment]
Changelog entry needed.

**Comment by floion** [comment]
@telphan the poky submodule should be updated to tag rocko-18.0.2

File: layers/meta-resin-jetson/conf/layer.conf [hunk 1]
```diff
@@ -6,3 +6,5 @@ BBFILES += "${LAYERDIR}/recipes-*/*/*.bb \
 BBFILE_COLLECTIONS += "resin-jetson"
 BBFILE_PATTERN_resin-jetson := "^${LAYERDIR}/"
 BBFILE_PRIORITY_resin-jetson = "1337"
+
+GCCVERSION = "6.%"
```
Comment (floion) [inline layers/meta-resin-jetson/conf/layer.conf]: What are the errors with gcc7?

Comment (telphan) [reply layers/meta-resin-jetson/conf/layer.conf]: https://github.com/madisongh/meta-tegra/issues/59

**Pull request merged**